docs/developer/style.md

```diff
@@ -100,121 +96,171 @@ for _, tt := range tests {
 ```
 
 </td>
+<td>
+
+```go
+test.New(
+    test.WithCase(
+        caser.New(
+            WithName("case_1"),
+            WithArg("192.0.2.0:8000")
+            WithWant("192.0.2.0", "8000", nil),
```

Comment:
do we need `nil` here?

Reply:
Because `SplitHostPort` returns 3 objects (host, port, error), I think that `nil` is necessary.
`nil` is used by compares of error.

Reply:
oh sorry I missed it. thank you for your answer :)